tests/StructureDiscovererTest.php

```diff
@@ -277,6 +278,54 @@ public function satisfies(DiscoveredStructure $discoveredData): bool
     ]);
 });
 
+it('can sort discovered files', function (
+    Sort $sort,
+    array $expected,
+) {
+    $found = Discover::in(__DIR__ . '/Fakes')->classes()->sortBy($sort)->get();
+
+    expect($found)->toEqual($expected);
+})->with(
+    [
+        'name' => [
+            Sort::Name,
```

Comment:
I only included tests for sorting by the file name, as some of the other sort options could change depending on the contents of the file (size), the timestamps (modified, changed, accessed).

If you'd like me to create file stubs for the other testable options like `type` and `size`, let me know! 👍

Reply:
What we have now is fine, thanks!